sequence_transform.c

```diff
@@ -243,6 +255,45 @@ void log_rule(st_trie_t *trie, st_trie_search_result_t *res) {
     // Terminator
     uprintf("\n");
 }
+//////////////////////////////////////////////////////////////////////
+__attribute__((weak)) void sequence_transform_on_missed_rule_user(const st_trie_rule_t *rule)
+{
+    uprintf("Missed rule! %s -> %s\n", rule->sequence, rule->transform);
+}
+//////////////////////////////////////////////////////////////////////
+void st_find_missed_rule(void)
+{
+    char sequence_str[SEQUENCE_MAX_LENGTH + 1] = {0};
+    char transform_str[TRANSFORM_MAX_LEN + 1] = {0};
```

Comment:
These aren't big enough if they hold multi-byte utf8 characters, and I believe the stars are multi-byte characters.